tools/restatectl/src/commands/log/find_tail.rs

```diff
@@ -31,28 +30,23 @@ pub struct FindTailOpts {
 }
 
 async fn find_tail(connection: &ConnectionInfo, opts: &FindTailOpts) -> anyhow::Result<()> {
-    let channel = grpc_channel(connection.cluster_controller.clone());
-    let mut client =
-        ClusterCtrlSvcClient::new(channel).accept_compressed(CompressionEncoding::Gzip);
-
     let mut chain_table = Table::new_styled();
     let header_row = vec!["LOG-ID", "SEGMENT", "STATE", "TAIL-LSN"];
 
     chain_table.set_styled_header(header_row);
 
     for log_id in opts.log_id.clone().into_iter().flatten() {
         let find_tail_request = FindTailRequest { log_id };
-        let response = match client.find_tail(find_tail_request).await {
-            Ok(response) => response.into_inner(),
-            Err(err) => {
-                chain_table.add_row(vec![
-                    Cell::new(log_id),
-                    Cell::new(err.code()).fg(Color::DarkRed),
-                    Cell::new(err.message()).fg(Color::DarkRed),
-                ]);
-                continue;
```

Comment:
Is it intended to remove this logic if cannot find the tail for a given `log_id`?

Reply:
It is intended. I can partially restore it